docs/modifying-akri-installation.md

```diff
@@ -45,11 +48,12 @@ command could be run:
 ```bash 
 helm upgrade akri akri-helm-charts/akri-dev \
     --set useLatestContainers=true \
-    --set onvifVideo.enabled=true \
-    --set onvifVideo.ipAddresses.action=Exclude \
-    --set onvifVideo.ipAddresses.items[0]=10.0.0.1
+    --set onvif.enabled=true \
+    --set onvif.brokerPod.image.repository="ghcr.io/../onvif-broker" \
```

Comment:
why ghcr.io/../onvif-broker?  should '..' be explained somewhere?

Reply:
Good point. Do you think it is better to include the image -- maybe just adding \<your broker image> -- and explain how a broker image is needed for broker pods to be deployed or to not include it at all? and just focus on discovery?

Reply:
Made some changes to explain the broker images